### PR TITLE
Fixing IPoE and PPPoE documentation

### PR DESCRIPTION
**Pull request by aapostoliuk**
Fixing IPoE and PPPoE documentation

### CODE REVIEW

File: docs/configuration/service/ipoe-server.rst [hunk 1]
```diff
@@ -52,7 +52,7 @@ the configuration.
 
 .. cfgcmd:: set service ipoe-server authentication interface <interface> mac <MAC>
 
-    Creates local pppoe user with username=**<interface>** and
+    Creates local ipoe user with username=**<interface>** and
```
Comment (sever-sever) [inline docs/configuration/service/ipoe-server.rst:55]: There should be `IPoE` ie abbr, the same for `PPPoE` or `L2TP`

File: docs/configuration/service/ipoe-server.rst [hunk 2]
```diff
@@ -398,6 +389,11 @@ Global Advanced options
 Monitoring
 **********
 
+.. opcmd:: show ipoe-server sessions
+
+   Use this command to locally check the active sessions in the IPoe
```
Comment (sever-sever) [inline docs/configuration/service/ipoe-server.rst:394]: should be `IPoE`

File: docs/configuration/service/ipoe-server.rst (hunk 1 above)
Comment (aapostoliuk) [reply docs/configuration/service/ipoe-server.rst]: done

File: docs/configuration/service/ipoe-server.rst (hunk 2 above)
Comment (aapostoliuk) [reply docs/configuration/service/ipoe-server.rst]: done

**Review by sever-sever — APPROVED** [review]

**Comment by rebortg** [comment]
@mergifyio backport sagitta

**Pull request merged**